Gui/GUIutils/guiUtils.py

```diff
@@ -352,6 +357,66 @@ def CheckXMLValue(pFilename, pAttribute):
 ##########################################################################
 
 
+def GenerateXMLConfig(firmwareList, testName, outputDir, **arg):
+    outputFile = outputDir + "/CMSIT_" + testName + ".xml"
+    fileNanme = "/CMSIT_" + testName + ".xml"
+    filePath = outputDir
+    #AllModules = firmwareList.getAllModules().values() # orginal code for doing the OG setup, but i ingore this section
+
+    #ignore the section of setting up OGModule
+
+    #in the old code boardID is always zero, we might need to change it later when using multiple boards
+    #create the code for doing the summer first 
+    boardID = "0"
```

Comment:
boardID is the BeBoard Id in XML file. I believe it is unique for each board. What do you mean by using multiple boardID?

Reply:
We can double check with Matt, but I believe that BeBoard represents the FC7. We will soon be using two FC7s to test modules, the first one will have boardID = 0 and the second will have boardID=1. Once we start using two FC7s we can't set the boardID to 0, otherwise that'll cause problems with the second FC7.